src/main/java/io/ebean/util/StringHelper.java

```diff
@@ -68,14 +68,15 @@
       // dp("pos="+pos+" equalsPos="+equalsPos+"
       // endQuotePos="+endQuotePos);
       String name = tag.substring(pos, equalsPos);
-      String value = tag.substring(equalsPos + 2, endQuotePos);
       // dp("name="+name+"; value="+value+";");
 
       // trim off any whitespace from the front of name...
       name = trimFront(name, " ");
       if ((name.indexOf(SINGLE_QUOTE) > -1) || (name.indexOf(DOUBLE_QUOTE) > -1)) {
         throw new RuntimeException("attribute name contains a quote [" + name + "]");
       }
+
+      String value = tag.substring(equalsPos + 2, endQuotePos);
```

Comment:
Moved this here, past the throw statement. No point in creating the variable in memory and then potentially exiting the context without the variable ever having been used.